vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -678,4 +687,14 @@ Mso::CntPtr<IReactInstanceInternal> MakeReactInstance(
       reactHost, std::move(options), std::move(whenCreated), std::move(whenLoaded), std::move(updateUI));
 }
 
+#ifdef PATCH_RN
+#if defined(USE_V8)
+std::string ReactInstanceWin::getApplicationLocalFolder() {
+  auto local = winrt::Windows::Storage::ApplicationData::Current().LocalFolder().Path();
+
+  return std::wstring_convert<std::codecvt_utf8<wchar_t>>().to_bytes(std::wstring(local.c_str(), local.size())) + "\\";
```

Comment:
>std::wstring_convert<std::codecvt_utf8<wchar_t>>().to_bytes [](start = 9, length = 59)

This is deprecated in C++17. Could you use the winrt base.h conversion utilities instead?

Reply:
There are also already Unicode conversion utilities under Common\unicode.h. These are implemented on top of MultiByteToWideChar and WideCharToMultiByte.

---
In reply to: [387941980](https://github.com/microsoft/react-native-windows/pull/4234#discussion_r387941980) [](ancestors = 387941980)

Reply:
FWIW, I just copied this code from UwpReactInstance.cpp for parity, but will take a look.

---
In reply to: [387955340](https://github.com/microsoft/react-native-windows/pull/4234#discussion_r387955340) [](ancestors = 387955340,387941980)